antenna-documentation/src/site/markdown/template-bundle-development.md

```diff
@@ -4,6 +4,24 @@ automatically when no other bundle is configured. This guide will show how to cr
 integrate it in the Antenna workflow run. 
 
 ## Custom Template Bundle
+
+### How to create the template PDF files
+
+First, the template has to be developed with a tool like Microsoft Word / Open Office / Libre Office. The corresponding 
+document will consist of 4 pages which will represent the 4 templates PDF files:
+* cover.pdf
+* copyright.pdf 
+* content.pdf 
+* back.pdf
+
+The page size is A4 at 72 PPI with 595x842 pixels. The cover.pdf file will be the cover page of the attribution document 
+which will contain the project name and version starting at 80x547. The copyright.pdf file will be the second page of 
+the attribution document which will contain the project copyright notes starting at 70x659. The content.pdf file will be
+used several times which will be list the used dependencies with PURL, file name, license and copyright notes. The 
```

Comment:
Is there also a pixel position for this file?

Reply:
Yes, it's written on the page at 40,80 until the page bottom margin is reached.

Reply:
Please add this bit to the docs then.